datadog_checks_dev/datadog_checks/dev/tooling/commands/docs/deploy.py

```diff
@@ -0,0 +1,87 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+import click
+
+from ....utils import chdir, copy_path, create_file, dir_exists, path_join, temp_dir
+from ...constants import get_root
+from ...git import get_git_email, get_git_user, get_latest_commit_hash
+from ..console import CONTEXT_SETTINGS, abort, echo_info, echo_success, echo_waiting, echo_warning, run_or_abort
+
+PRODUCTION_BRANCH = 'gh-pages'
+
+
+@click.command(context_settings=CONTEXT_SETTINGS, short_help='Deploy built documentation')
+@click.argument('branch', required=False)
+@click.option('--yes', '-y', is_flag=True)
+@click.pass_context
+def deploy(ctx, branch, yes):
+    """Deploy built documentation."""
+    # We allow specifying a branch in case you want to quickly showcase changes to others
+    if not branch:
+        branch = PRODUCTION_BRANCH
```

Comment:
Just to clarify: for this use case we'd ship the `.zip` file, right? AFAIK GH Pages does not support deploying to multiple branches.

```suggestion
    # We allow specifying a branch in case you want to quickly showcase changes to others
    # In that case, you should ship them the zip archive linked to at the end of the script.
    if not branch:
        branch = PRODUCTION_BRANCH
```